docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,18 @@
 title: Changelog
 ---
 
+## 12.17.3
+
+_Released 08/01/2023_
+
+**Bugfixes:**
+
+- Fixed an issue where unexpected branch names were being recorded for cypress runs when executed by GitHub Actions. The HEAD branch name will now be recorded by default for pull request workflows if a branch name cannot otherwise be detected from user overrides or from local git data. Fixes [#27389](https://github.com/cypress-io/cypress/issues/27389).
```

Comment:
```suggestion
- Fixed an issue where unexpected branch names were being recorded for cypress runs when executed by GitHub Actions. The `HEAD` branch name will now be recorded by default for pull request workflows if a branch name cannot otherwise be detected from user overrides or from local git data. Fixes [#27389](https://github.com/cypress-io/cypress/issues/27389).
```